app/assets/onepager/js/send.js

```diff
@@ -349,4 +351,27 @@ var etherscanDomain = function() {
     // mainnet
   }
   return etherscanDomain;
+};
+
+var check_balance_and_alert_user_if_not_enough = function(tokenAddress, amount) {
```

Comment:
any way we can DRY (dont repeat yourself) with https://github.com/gitcoinco/web/blob/master/app/assets/v2/js/pages/new_bounty.js#L431-L448

Reply:
I would suggest the same.